### PR TITLE
Add/voxelands

### PR DESCRIPTION
**Pull request by PyroLagus**
Successfully tested. Depends on #7863

### CODE REVIEW

**Comment by geerds** [comment]
This PR also updates minetest

File: pkgs/games/voxelands/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ];
+
+  buildInputs = [
+    cmake irrlicht libpng bzip2 libjpeg sqlite
+    libXxf86vm mesa openal libvorbis x11 pkgconfig
+  ];
+
+  meta = {
+    homepage = "http://voxelands.com/";
+    description = "Infinite-world block sandbox game based on Minetest.";
```
Comment (geerds) [inline pkgs/games/voxelands/default.nix]: [Don’t include a period at the end.](http://nixos.org/nixpkgs/manual/)

File: pkgs/games/voxelands/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    license = stdenv.lib.licenses.gpl3Plus;
+  };
```
Comment (geerds) [inline pkgs/games/voxelands/default.nix]: Missing platforms attribute? (e.g. `platforms.linux`)

File: pkgs/games/voxelands/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG"
+  ];
+
+  buildInputs = [
+    cmake irrlicht libpng bzip2 libjpeg sqlite
+    libXxf86vm mesa openal libvorbis x11 pkgconfig
+  ];
+
+  meta = {
+    homepage = "http://voxelands.com/";
```
Comment (pSub) [inline pkgs/games/voxelands/default.nix]: You don't need quotes around urls.

**Comment by C0DEHERO** [comment]
All fixed up. I'm really starting to like git rebase :dancer:

**Comment by geerds** [comment]
Thank you very much for your work :+1: I've merged it manually and added myself to the maintainers

**Pull request closed without merging**